### PR TITLE
To execute the tests on rhel-9

### PR DESCRIPTION
**Pull request by SrinivasaBharath**
# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [x] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: pipeline/metadata/5.2.yaml [hunk 1]
```diff
@@ -255,7 +255,7 @@
 - name: "Testing RADOS basic regression scenarios"
   suite: "suites/pacific/rados/tier-2_rados_basic_regression.yaml"
   global-conf: "conf/pacific/rados/7-node-cluster.yaml"
-  platform: "rhel-8"
+  platform: "rhel-9"
   rhbuild: "5.2"
   inventory:
     openstack: "conf/inventory/rhel-8-latest.yaml"
```
Comment (pdhiran) [inline pipeline/metadata/5.2.yaml:261]: Platform file needs to be modified to ` rhel-9-latest.yaml `

File: pipeline/metadata/5.2.yaml [hunk 2]
```diff
@@ -303,7 +303,7 @@
 - name: "RADOS regression testing for stretched Clusters"
   suite: "suites/pacific/rados/tier-2_rados_test-stretch-mode.yaml"
   global-conf: "conf/pacific/rados/11-node-cluster.yaml"
-  platform: "rhel-8"
+  platform: "rhel-9"
   rhbuild: "5.2"
   inventory:
     openstack: "conf/inventory/rhel-8-latest.yaml"
```
Comment (pdhiran) [inline pipeline/metadata/5.2.yaml:309]: same as above

**Comment by pdhiran** [comment]
Can we have one run log added?

File: pipeline/metadata/5.2.yaml (hunk 1 above)
Comment (SrinivasaBharath) [reply pipeline/metadata/5.2.yaml:261]: yes.. I will do

Comment (SrinivasaBharath) [reply pipeline/metadata/5.2.yaml]: Updated.

File: pipeline/metadata/5.2.yaml (hunk 2 above)
Comment (SrinivasaBharath) [reply pipeline/metadata/5.2.yaml]: Updated.

**Review by pdhiran — APPROVED** [review]
Looks Good

**Review by tintumathew10 — APPROVED** [review]
LGTM

**Pull request merged**